packages/dashboard/src/components/dashboard/index.tsx

```diff
@@ -43,6 +43,7 @@ export type DashboardProperties = {
   name?: string;
   onViewportChange?: ViewportChange;
   currentViewport?: Viewport;
+  timeZone?: string;
```

Comment:
is timezone a saveable prop or can it just be changed for on the fly analysis? 

if its saved by the dashboard `onSave` action then maybe it should belong in the dashboardConfiguration object since that's how we handle other saveable configs (for example the initialViewport and canvas sizes are saved in this config)

Reply:
Good call I think this should be saved, will update.

Reply:
After thinking about it more, I dont think we need to save this prop. The intention is that if this prop is provided in the dashboard component, then the charts/components within the dashboard will be displayed in the provided timezone. There is no option to change this timezone in the UI. So I don't think it needs to be part of `onSave`.